server/api.js

```diff
@@ -164,4 +100,39 @@ router.get("/bidder-bid", async (req, res) => {
 			.send({ code: "SERVER_ERROR" });
 });
 
+router.get("/tenders", async (req, res) => {
+	const page = parseInt(req.query.page, 10) || 1;
+	const limit = parseInt(req.query.limit, 10) || 25;
+	const offset = (page - 1) * limit;
+
+	const countSql = "SELECT COUNT(*) FROM tender";
+	const dataSql = `
+		SELECT id, title, creation_date, announcement_date, deadline, status 
+		FROM tender 
+		ORDER BY creation_date DESC 
+		LIMIT $1 OFFSET $2
+	`;
+	try {
+		const countResult = await db.query(countSql);
+		const totalItems = parseInt(countResult.rows[0].count, 10);
+		const totalPages = Math.ceil(totalItems / limit);
+
+		const dataResult = await db.query(dataSql, [limit, offset]);
+		const tenders = dataResult.rows;
+
+	res.status(200).json({
+		resource: {
+			tenders,
+			pagination: {
+				itemsPerPage: limit,
+				currentPage: page,
+				totalPages,
+			},
+		},
```

Comment:
This doesn't follow the standard I put in #launch-module-web-wizards. Please verify it